class-components/.editorconfig

```
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+max_line_length = off
+trim_trailing_whitespace = false
```

```suggestion
trim_trailing_whitespace = false

```
добавляй в таких местах пустые строки в конце чтоб не ругался гитхаб